charts/rstudio-workbench/README.md

```diff
@@ -33,27 +33,27 @@ This chart requires the following in order to function:
 
 * A license key, license file, or address of a running license server. See the `license` configuration below.
 * A Kubernetes [PersistentVolume](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) that contains the home directory for users.
-  * If `homeStorage.create` is set, a PVC that relies on the default storage class will be created to generate the PersistentVolume. 
-    Most Kubernetes environments do not have a default storage class that you can use with `ReadWriteMany` access mode out-of-the-box. 
-    In this case, we recommend you disable `homeStorage.create` and  create your own `PersistentVolume` and `PersistentVolumeClaim`, then mount them 
+  * If `homeStorage.create` is set, a PVC that relies on the default storage class will be created to generate the PersistentVolume.
+    Most Kubernetes environments do not have a default storage class that you can use with `ReadWriteMany` access mode out-of-the-box.
+    In this case, we recommend you disable `homeStorage.create` and  create your own `PersistentVolume` and `PersistentVolumeClaim`, then mount them
     into the container by specifying the `pod.volumes` and `pod.volumeMounts` parameters.
```

Comment:
```suggestion
    into the container by specifying the `pod.volumes` and `pod.volumeMounts` parameters, or by specifying your `PersistentVolumeClaim` using `sharedStorage.mount`.
```

This is getting to be a complicated flow chart of a description, but do you think we should include this information? If so, we'll want to update in all three product charts.

Reply:
I also mentioned `sharedStorage.name` since you need both that and `sharedStorage.mount`. I updated this PR for RSW, and I pushed a change for RSC and RSPM into #22.